src/Microsoft.AspNet.Identity/UserManager.cs

```diff
@@ -1929,17 +1949,23 @@ public virtual Task<IList<TUser>> GetUsersInRoleAsync(string roleName)
         /// <param name="user"></param>
         /// <param name="methodName"></param>
         /// <returns>result</returns>
-        protected async Task<bool> LogResultAsync(bool result,
+        protected virtual async Task<bool> LogResultAsync(bool result,
             TUser user, [System.Runtime.CompilerServices.CallerMemberName] string methodName = "")
         {
-            var baseMessage = Resources.FormatLoggingResultMessage(methodName, await GetUserIdAsync(user));
-            if (result)
-            {
-                Logger.WriteInformation(string.Format("{0} : {1}", baseMessage, result.ToString()));
-            }
-            else
+            var logLevel = result ? LogLevel.Information : LogLevel.Warning;
+
+            if (Logger.IsEnabled(logLevel))
             {
-                Logger.WriteWarning(string.Format("{0} : {1}", baseMessage, result.ToString()));
+                var baseMessage = Resources.FormatLoggingResultMessage(methodName, await GetUserIdAsync(user));
+
+                if(logLevel == LogLevel.Warning)
```

Comment:
Is there a version of `Write` we can use from logging that would just take `logLevel` as a parameter?